android/src/main/java/co/airbitz/fastcrypto/MoneroAsyncTask.java

```diff
@@ -91,7 +91,10 @@ protected Void doInBackground(Void... voids) {
                     promise.resolve(out);
                 }
             } catch (Exception e) {
-                promise.reject("Err", e);
+                String detailedError = String.format(
+                        "{\"err_msg\":\"Exception occurred: %s\"}",
+                        e.getMessage() != null ? e.getMessage() : "Unknown exception");
+                promise.resolve(detailedError);
```

Comment:
Shouldn't this reject?

Reply:
Nom It should not reject. We need to get the `err_msg` to make it more descriptive for debugging. 
If we use reject, the error will be caught [here](https://github.com/ExodusMovement/assets/blob/522a94749881ea4dd45037da7ca6f0f93c7e134c/monero/monero-client/src/monero-interface.js#L15-L17) and hidden